compile-all

```diff
@@ -795,8 +795,8 @@ if [ "false" == "$BASEONLY" ]; then
 	
 	echo "$0: ==== Installing GNUstep Gorm..."
 	if [ "$UNAME" == "MINGW64_NT" ] ; then
-# Switch MSYS to use emulated symbolic links so that tar can create files with dangling links during make install
-	   MSYS=winsymlinks:lnk ; export MSYS
+# Unset MSYS so that tar can create files with dangling links during make install
+	   #unset MSYS
```

Comment:
Is this one supposed to be commented out? It causes a syntax error while the script is running.

Reply:
No, it should not be commented out.  I have not been able to build GNUstep Gorm on my system.  So, I have been commenting out the section building the Gorm app.  Apparently, I did not uncomment this in the Gorm app build section after my last run, before committing.  I will push another commit to uncomment this "unset MSYS".
When "make install" is successful, you should see a DLL update under C:\msys64\usr\local\GNUstep\Local\Library\Frameworks\ for CHDataStructures and/or MAKeyedArchiver.